HMCL/src/main/resources/assets/lang/I18N_zh.properties

```
@@ -666,6 +666,39 @@ message.confirm=提示
 message.copied=已複製到剪貼簿
 message.default=預設
 message.doing=請耐心等待
+
+message.did_you_know=你知道嗎？
+message.tips_1=HMCL 最初由 huanghongxun 開發，後來越來越多貢獻者加入其中。
+message.tips_2=Minecraft 的第一個版本於 2009 年推出，當時只是個簡單的原型。
+message.tips_3=在 Minecraft 中，每匹馬的速度都不同，這些數值皆為隨機生成。
+message.tips_4=將衛道士命名為 Johnny 會讓牠變得極度暴躁。
+message.tips_5=HMCL 的開發者社群非常活躍，經常推出新功能與改進，歡迎到 GitHub 追蹤討論。
+message.tips_6=HMCL 支援多平台，包括 Windows、macOS 與 Linux。
+message.tips_7=HMCL 的開發者常在 B 站與 GitHub 社群分享開發進度與未來計畫。
+message.tips_8=鐵魔像會把罌粟送給附近的銅魔像，收到花的銅魔像會把它戴在頭上。
+message.tips_9=終界使者若被困在礦車或蜘蛛網中，便無法再瞬移。
+message.tips_10=苦力怕的原型其實是豬，因程式錯誤而誕生。
+message.tips_11=若你在遊戲中連續三日不眠，第四晚可要當心了。
+message.tips_12=用雞蛋一次砸出 4 隻小雞的機率是 1/256。
+message.tips_13=金製鎬子雖耐久極低，但附魔出好屬性的機率很高。
+message.tips_14=山貓與鐵魔像不會因摔落而受傷，因為牠們免疫摔落傷害。
+message.tips_15=摔落在乾草塊上可減少摔落傷害。
+message.tips_16=在早期 Minecraft 中，煤礦可以徒手直接挖。
+message.tips_17=聖誕節期間，箱子會變成禮物箱外觀。
+message.tips_18=貓會自己跳上床、箱子或熔爐坐著。
+message.tips_19=終界使者是原版 Minecraft 中唯一能在三個維度自然生成的生物。
+message.tips_20=旗幟在主世界會輕微飄動，在地獄與終界則靜止不動。
+message.tips_21=Minecraft 的基本時間單位為「刻」，1 刻 = 0.05 秒，一天 = 24000 刻 = 20 分鐘。
+message.tips_22=熱帶魚有 2 種體型，每種 6 種花紋，底色與花紋色各有 15 種，總共 2700 種組合。
+message.tips_23=將羊命名為 jeb_，牠會變成彩虹漸層色。
+message.tips_24=骷髏有 21% 機率以左手持弓，據說這是根據現實人類左撇子比例設定的。
+message.tips_25=骷髏的攻擊不具方向性，別以為背對牠就能躲過箭。
+message.tips_26=比較馬的速度：兩人各騎一匹馬，用拴繩相連後朝反方向拉，被拉動的那匹較慢。
+message.tips_27=攻擊魷魚的觸手不會造成傷害，瞄準頭部才有效。
+message.tips_28=成年海龜不會被不死生物攻擊，除非該不死生物在牠幼年時就已對牠產生敵意。
+message.tips_29=叢林神廟總是由 1185 個原石或苔蘚石組成。
```

建议遵循台湾地区的语言习惯。目前此文件内其他内容都是按照台湾繁体写的。

```suggestion
message.tips_29=叢林遺跡總是由 1185 個鵝卵石或青苔鵝卵石組成。
```

[青苔鵝卵石 - 中文 Minecraft Wiki](https://zh.minecraft.wiki/w/%E8%8B%94%E7%9F%B3?variant=zh-tw)